aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/PointsAboveInstance.java

```diff
@@ -21,17 +21,19 @@
 
 package com.spotify.heroic.aggregation.simple;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.EqualsAndHashCode;
-import lombok.ToString;
+import lombok.Data;
 
-import java.beans.ConstructorProperties;
-
-@ToString
+@Data
 @EqualsAndHashCode(callSuper = true)
 public class PointsAboveInstance extends MetricMappingAggregation {
+    private final double threshold;
 
-    @ConstructorProperties({"threshold"})
```

Comment:
So we actually want to make use of `ConstructorProperties` here. The problem was (I think) that mapping was being serialized which you have already prevented. So this change should not be necessary.

Reply:
I didn't see it called on the AboveKInstance. Just trying to be consistent. I'm happy to add it back in, just not too sure what it does.

Reply:
Yeah, please do. Using `ConstructorProperties` is a standard annotation and is currently preferred. We'll probably do a small project getting rid of the `JsonCreator` et. al. in the future.